src/main/kotlin/no/nav/klage/oppgave/domain/kodeverk/Kilde.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.klage.oppgave.domain.kodeverk
+
+enum class Kilde {
+    OPPGAVE, MODERNE
```

Comment:
Har nok ikke tenkt så langt på dette, men tror jeg så for meg at kilde skulle si noe om hvor klagebehandlingen kom fra. Men vi kanskje har mer behov av å vite om det er modernt eller fra oppgave?

Reply:
Mulig jeg skulle kalt det noe annet enn MODERNE.. ? KAFKA_V1 f.eks?

Reply:
Det blir jo omtrent det samme. Tenkte kanskje at kilde kunne være en eller anne kode for et gitt fagsystem f.eks. Men vi kan se hva blir bra der.